src/Backend/OnboardingInsuranceAPI/Areas/Insurance/CarInsuranceData.cs

```diff
@@ -0,0 +1,25 @@
+using System.Text.Json.Serialization;
+
+namespace OnboardingInsuranceAPI.Areas.Insurance;
+
+public record CarInsuranceData
+{
+    [JsonPropertyName("expDate")]
+    public string ExpDate { get; init; } //Expiration date
+    [JsonPropertyName("informationNote")]
+    public string InformationNote { get; init; } //Note about insurancr from agent
+    [JsonPropertyName("yearlyContribution")]
+    public decimal? YearlyContribution { get; init; }
+    [JsonPropertyName("carType")]
+    public string CarType { get; init; } //Type of car "for example sportcar"
```

Comment:
You could create Enum of car types, that then gets straight up converted from string into Enum on the backend.